libs/island-ui/core/src/lib/DatePicker/DatePicker.tsx

```diff
@@ -225,6 +225,7 @@ const CustomHeader = ({
         </Text>
       </div>
       <button
+        data-testid="datepickerIncreaseMonth"
```

Comment:
should this be more unique? and maybe be added to more critical elements inside the datepicker?
```suggestion
        data-testid={`${testId}-datepickerIncreaseMonth`}
```

Reply:
I think this only becomes an issue if you have two datepickers open at the same time and try to select this element. I think this is fine for now.